satpy/tests/test_composites.py

```diff
@@ -173,83 +175,165 @@ def setUp(self):
         self.ds4 = ds4
 
         # high resolution version - but too big
-        ds4 = xr.DataArray(da.ones((4, 4), chunks=2, dtype=np.float64),
-                           attrs=attrs.copy(), dims=('y', 'x'),
-                           coords={'y': [0, 1, 2, 3], 'x': [0, 1, 2, 3]})
-        ds4.attrs['name'] += '4'
-        ds4.attrs['resolution'] = 500
-        ds4.attrs['rows_per_scan'] = 1
-        ds4.attrs['area'] = AreaDefinition('test', 'test', 'test',
-                                           {'proj': 'merc'}, 4, 4,
-                                           (-2000, -2000, 2000, 2000))
-        self.ds4_big = ds4
+        ds4_big = xr.DataArray(da.ones((4, 4), chunks=2, dtype=np.float64),
+                               attrs=attrs.copy(), dims=('y', 'x'),
+                               coords={'y': [0, 1, 2, 3], 'x': [0, 1, 2, 3]})
+        ds4_big.attrs['name'] += '4'
+        ds4_big.attrs['resolution'] = 500
+        ds4_big.attrs['rows_per_scan'] = 1
+        ds4_big.attrs['area'] = AreaDefinition('test', 'test', 'test',
+                                               {'proj': 'merc'}, 4, 4,
+                                               (-2000, -2000, 2000, 2000))
+        self.ds4_big = ds4_big
 
-    def test_bad_color(self):
+    @pytest.mark.parametrize(
+        "exp",
+        [
+            ValueError,
+        ]
+    )
```

Comment:
This could be removed as-is since there is only one parameter. You could combine these two "bad color" tests where you have parameters `("init_kwargs", "exp_exception")`. Actually, it is `ValueError` in both cases so I guess it could be removed in that case too.